xchange-coindeal/src/main/java/org/knowm/xchange/coindeal/dto/trade/CoindealTradeHistory.java

```diff
@@ -0,0 +1,159 @@
+package org.knowm.xchange.coindeal.dto.trade;
+
+import com.fasterxml.jackson.annotation.*;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import java.util.HashMap;
+import java.util.Map;
+
+//@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "id",
+        "clientOrderId",
+        "orderId",
+        "symbol",
+        "side",
+        "quantity",
+        "fee",
+        "price",
+        "timestamp"
+})
```

Comment:
JsonPropertyOrder - I don't think this is necessary to properly deserialize the object plus serialization would yous the order in which you declared your properties anyway

Reply:
OK, i delete it